test/langtools/jdk/javadoc/doclet/testNewLanguageFeatures/pkg1/B.java

```diff
@@ -36,7 +36,8 @@
    a = @AnnotationType(optional="foo",required=1994),
    sa = {"up", "down"},
    primitiveClassTest = boolean.class,
-   arrayClassTest = String[].class,
-   arrayPrimitiveTest = boolean[].class)
+   arrayClassTest = String[][].class,
+   arrayPrimitiveTest = boolean[].class,
+   classArrayTest = {TypeParameterSubClass[].class, String.class, long[][][].class})
```

Comment:
Consider adding multi-dimensional array of `TypeParameterSubClass`, to test link generation. The problem with `long[][][]` is that there's nowhere to link to. The problem with `String[][]` is that JDK source where the `java.lang.String` doc comment resides is unavailable.

Alternatively, you could probably link to platform, so that we could see output for `String[][]`.